build.ps1

```diff
@@ -71,16 +73,18 @@ foreach ($profile in $profiles) {
  
   Write-Host "Building Orbit in build_$profile/ with conan profile $profile"
 
-  & $py.Path -3 -m conans.conan install -if build_$profile\ --build outdated -pr:b $build_profile -pr:h $profile -u "$PSScriptRoot"
+  conan install -if build_$profile\ --build outdated "-pr:b" $build_profile "-pr:h" $profile -u "$PSScriptRoot"
 
   if ($LastExitCode -ne 0) {
     Throw "Error while running conan install."
   }
-
-  $process = Start-Process $py.Path -NoNewWindow -ErrorAction Stop -PassThru -ArgumentList "-3","-m","conans.conan","build","-bf","build_$profile/",$PSScriptRoot
-  $handle = $process.Handle # caching handle needed due to bug in .Net
-  $process.WaitForExit()
-  if ($process.ExitCode -ne 0) {
+  
+  conan build -bf "build_$profile/" $PSScriptRoot
+  if ($LastExitCode -ne 0) {
     Throw "Error while running conan build."
   }
+
+  if ($profile.StartsWith("default_")) {
+    Write-Host "The build finished successfully. Start Orbit with .\build_$profile\bin\Orbit!"
```

Comment:
Why is this guarded by the default profile?

Reply:
I consider this message noise for the daily development workflow, but helpful for newcomers. That's why I thought a good compromise would be to only display it when using a default profile (which is not recommended for development and the default when calling bootstrap without arguments).

I can also remove the message though if you think that introduces too much inconsistency - or move it to bootstrap-orbit.*

Reply:
Makes sense. Let's keep it as you have it in your change.